src/main/java/io/github/reoseah/toxsky/ToxSky.java

```diff
@@ -32,10 +34,10 @@
 public class ToxSky implements ModInitializer {
     public static final Logger LOGGER = LoggerFactory.getLogger("toxsky");
 
-    public static final Block FLOATING_GARBAGE = new FloatingGarbageBlock(Block.Settings.create());
-    public static final Block GARBAGE_BLOCK = new Block(Block.Settings.create().strength(2F));
-    public static final Block SEAWEED_GARBAGE_BLOCK = new Block(Block.Settings.create().strength(2F));
-    public static final Block RECYCLED_PLASTIC_BLOCK = new Block(Block.Settings.create().strength(2F));
+    public static final Block FLOATING_GARBAGE = new FloatingGarbageBlock(Block.Settings.create().mapColor(MapColor.LIGHT_BLUE));
+    public static final Block GARBAGE_BLOCK = new Block(Block.Settings.create().mapColor(MapColor.PINK).strength(2F));
+    public static final Block SEAWEED_GARBAGE_BLOCK = new Block(Block.Settings.create().mapColor(MapColor.GREEN).strength(2F));
+    public static final Block RECYCLED_PLASTIC_BLOCK = new Block(Block.Settings.create().mapColor(DyeColor.ORANGE).strength(2F));
```

Comment:
I would use either `MapColor` or `DyeColor` for all blocks, not mixing them

Reply:
Otherwise looks good, thanks and sorry for delayed response

Hopefully I don't forget to check it when at home for another week ^^